### PR TITLE
Get management interface port number from dappvpn.config.json for Client.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-507.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -89,6 +95,60 @@ func getPWDStorage(conf *config) data.PWDGetSetter {
...
+	port, err := strconv.ParseUint(params[1], 10, 16)
+	if err != nil {
+		logger.Warn("port for OpenVpn management"+
+			" interface is wrong: %s", err)
+		return 0
+	}
+	return uint16(port)
+}
+
 func main() {
```
Comment (furkhat) [inline main.go]: I'm worried of our main.go becoming this big. If you can separate it nicely, please do.

File: main.go [hunk 2]
```diff
@@ -89,6 +95,60 @@ func getPWDStorage(conf *config) data.PWDGetSetter {
 	return &storage
 }
 
+func getVpnManagementPort(conf *config, logger *util.Logger, db *reform.Querier) uint16 {
+	if agent, err := data.IsAgent(db); err != nil || agent {
```
Comment (furkhat) [inline main.go]: This functions has lots of context to understand it. Please add explanatory comments.

File: main.go (hunk 1 above)
Comment (IntegralTeam) [reply main.go]: update

Comment (dzeckelev) [reply main.go]: Updated.

File: client/svcrun/misc.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	"github.com/privatix/dappctrl/data"
+	"github.com/privatix/dappctrl/svc/dappvpn/mon"
+	"github.com/privatix/dappctrl/util"
+)
+
+// ManagementPortFromConfiguration returns OpenVpn management
+// interface port from configuration.
+func ManagementPortFromConfiguration(conf *Config, service string,
+	logger *util.Logger,
+	db *reform.Querier) uint16 {
```
Comment (ababo) [inline client/svcrun/misc.go]: Please remove a line break here.

File: client/svcrun/misc.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
+package svcrun
```
Comment (ababo) [inline client/svcrun/misc.go]: This is wrong since this package is abstracted from concrete service types. Management port is a `VPN`-specific notion.

Comment (dzeckelev) [reply client/svcrun/misc.go]: Moved back to `main` pkg.

**Pull request closed without merging**